worlds/hylics2/__init__.py

```diff
@@ -109,6 +126,22 @@ def generate_basic(self):
                 .place_locked_item(self.add_item(gestures[200688]["name"], gestures[200688]["classification"], 200688))
             self.multiworld.get_location("Sage Airship: TV", self.player)\
                 .place_locked_item(self.add_item(gestures[200685]["name"], gestures[200685]["classification"], 200685))
+            self.pre_fill_items.append(\
```

Comment:
since this entire item creation is a list, it would be significantly faster if you did ```self.pre_fill_items += [
self.add_item(),
self.add_item(),
]```
etc.